tests/test_stft.py

```diff
@@ -0,0 +1,37 @@
+import torch
+import numpy as np
+from demucs.spec import spectro
+
+def spectroAPI(x, n_fft=512, hop_length=None, pad=0):
+    return NotImplementedError("This function is just a placeholder for the spectro function.")
+
+def test_spectrogram_shape_pytorch():
+    """ Test the shape of the spectrogram output using PyTorch's STFT """
+    batch, channels, samples = 1, 2, 2*44100
+    n_fft, hop_length = 4096, 1024
+    waveform = torch.randn(batch, channels, samples)  # (batch, channels, samples)
+    spec = spectro(waveform, n_fft=n_fft, hop_length=hop_length, pad=0, stft_type='pytorch')
```

Comment:
You can use Pytest Fixture here so the waveform generation can be reused

```suggestion
@pytest.fixture(scope='session')
def 20_second_waveform:
    batch, channels, samples = 1, 2, 2*44100
    n_fft, hop_length = 4096, 1024
    yield torch.randn(batch, channels, samples)  # (batch, channels, samples)

def test_spectrogram_shape_pytorch(20_second_waveform):
    """ Test the shape of the spectrogram output using PyTorch's STFT """
    spec = spectro(20_second_waveform, n_fft=n_fft, hop_length=hop_length, pad=0, stft_type='pytorch')
```

`20_second_waveform` can also be used for `test_spectrogram_shape_custom`